FTB/Signatures/test_CrashInfo.py

```diff
@@ -20,6 +21,8 @@
     NoCrashInfo, MinidumpCrashInfo, AppleCrashInfo, CDBCrashInfo, RustCrashInfo
 from FTB.Signatures.CrashSignature import CrashSignature
 
+CWD = os.path.dirname(__file__)
```

Comment:
Afaik, this needs to be `os.path.dirname(os.path.realpath(__file__))` instead to properly cope with relative paths and/or symlinks.

Reply:
Possibly, yeah. I'll merge and then push this fix to master.